### ISSUE TITLE
some skinning animations/vertices positions are not correctly imported

### ISSUE THREAD

**Issue by ghost** [KhronosGroup/glTF-Blender-IO#1875]
**Describe the bug**

some skinning animations/vertices positions are not correctly imported in blender. They simply become zero for some reason. 
@julienduroure could you take a look at the file ?

**Expected behavior**
![screenshot](https://user-images.githubusercontent.com/113032770/223385167-462e5379-49ab-4eb6-bc27-21febd85efe0.png)

**Screenshots**

![screenshot2](https://user-images.githubusercontent.com/113032770/223385179-9ac6802e-2a1d-4a81-a090-048e9102e0ce.png)

**.blend file/ .gltf**

[unrealexport.001.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/10909384/unrealexport.001.zip)
[unrealexport.002.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/10909371/unrealexport.002.zip)

The file is a bit large. Rename to `unrealexport.zip.001` and `unrealexport.zip.002` and then unzip.

**Version**
 - OS: Windows
 - Blender Version 3.3

**Comment by julienduroure** [comment]
We may have to have better error handling, but at a first look on:
https://github.khronos.org/glTF-Validator/

There are lots of error in glb file.
Seems this file was generated by Unreal. You should report it to Unreal or the addon creator.

**Comment by scurest** [comment]
Did you delete the file?

A bunch of spikes going to the origin is usually a sign of something up with the inverse bind matrices. See if toggling "Guess Original Bind Pose" does anything.

**Comment by scurest** [comment]
Wow, that is _a lot_ of errors. No wonder all the 3D formats without validators have such bad interoperability.

The vertices are at the origin, not because of inv binds, because their skinning weights are all zero (ie. they belong to no vertex groups). You can compare with a working vertex: see the vertex groups in the sidebar?

![Sidebar displays no vertex groups for the bad verts; compare to a normal vert](https://user-images.githubusercontent.com/11024420/223468256-e5a290ca-53f3-4861-9292-03970a01503d.png)

That's illegal in glTF. Either all the verts in a mesh have vertex groups, or none do. Being at the origin is the natural correct place for them in that case since their position is a weighted sum where all the weights are zero.

**Comment by scurest** [comment]
No, that would be complicated.

I looked up what Three is doing: it just [forces the first weight to 1 when the weights are all zero](https://github.com/mrdoob/three.js/blob/b98157b6f97060a2a75962adfe4c98ce837ab268/src/objects/SkinnedMesh.js#L123-L149).

I also looked at Babylon. In Babylon several polys are missing from the finger tip, so I suspect it's actually dividing by zero, resulting in Infinity, which I guess makes GL kill the poly or something?

I suppose it is possible for the importer to do the same thing Three does, that's up to julienduroure to decide.
